common/OpTestUtil.py

```diff
@@ -1082,6 +1082,52 @@ def ping_fail_check(self, i_ip):
             return False
         return True
 
+    def get_pci_type(self, pci_device, cv_HOST):
+        """
+        Get PCI device type
+        """
+        bus_id = pci_device[0:7]
+        device_file = "cat /sys/class/pci_bus/{}/device/{}/uevent".format(bus_id, pci_device)
```

Comment:
One extra line can be remove `bus_id = pci_device[0:7]` and directly use  ....format(pci_device[0:7], pci_device)